test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.IO.Pipelines;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Internal;
+using Microsoft.AspNetCore.Sockets;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+    public class HubEndpointTests
+    {
+        [Fact]
+        public async Task HubsAreDisposed()
+        {
+            var trackDispose = new TrackDispose();
+            var serviceProvider = CreateServiceProvider((s) => s.AddSingleton(trackDispose));
+
+            var endPoint = serviceProvider.GetService<HubEndPoint<TestHub>>();
+
+            using (var connectionWrapper = new ConnectionWrapper())
+            {
+                var endPointTask = Task.Run(() => endPoint.OnConnectedAsync(connectionWrapper.Connection));
```

Comment:
can't it be just var `endPointTask = endPoint.OnConnectedAsync(connectionWrapper.Connection);` ?